crates/sui-core/src/unit_tests/data/coin_deny_list_v2/sources/regulated_coin.move

```diff
@@ -17,6 +17,8 @@ module coin_deny_list_v2::regulated_coin {
             true,
             ctx
         );
+        let coin = coin::mint(&mut treasury_cap, 1000000, ctx);
+        transfer::public_transfer(coin, ctx.sender());
```

Comment:
What is this change?

Reply:
This is to support the new tests in `config_tests.rs`. So we can transfer regulated coins.